app/controllers/hiring_staff/vacancies/copy_controller.rb

```diff
@@ -23,12 +22,26 @@ def create
   private
 
   def copy_form_params
+    persist_nqt_job_role_to_nqt_attribute
```

Comment:
As there is this complexity here, it's probably worth adding something to `spec/features/hiring_staff_can_copy_a_vacancy_spec.rb` to ensure that copying a vacancy where the job_role includes nqt results in a new vacancy with newly_qualified_teacher == true.

Reply:
Separately, while you are touching `spec/features/hiring_staff_can_copy_a_vacancy_spec.rb`, I suspect we should update the lists of attributes expected to have been copied.

Reply:
> As there is this complexity here, it's probably worth adding something to `spec/features/hiring_staff_can_copy_a_vacancy_spec.rb` to ensure that copying a vacancy where the job_role includes nqt results in a new vacancy with newly_qualified_teacher == true.

Given this method is abstracted, and its behaviour is tested elsewhere, and the NQT field itself will be deleted soon, I would err against adding this to the feature spec

Reply:
> Separately, while you are touching `spec/features/hiring_staff_can_copy_a_vacancy_spec.rb`, I suspect we should update the lists of attributes expected to have been copied.

I don't think this is necessary